content/specs/versions-windows.md

```diff
@@ -49,3 +49,8 @@ weight: 8
 ## Android Studio 2020.3 (AI-203.7717.56.2031.7784292)
 
 Android Studio path: `C:\Program Files\Android\Android Studio\`
+
+## Unity 2020.3.23f1
+
+`UNITY_PATH` environment variable is set to `C:\Program Files\Unity\Hub\Editor\2020.3.23f1\Editor\Unity.exe`.
```

Comment:
actually we have a typo in previous doc
the var name should be `UNITY_HOME` and point to the folder

let's change in both doc files

Reply:
for extra context 
https://docs.codemagic.io/yaml-quick-start/building-a-unity-app/#license-activation